agent/util-scripts/pbench-register-tool

```diff
@@ -354,13 +362,13 @@ for (( i=0; ${i} < ${#remotes_A[@]}; i++ )); do
 	for local_interface in ${local_interfaces}; do
 		if [[ "${remote}" == "${local_interface}" ]]; then
 			if [[ -z "${local_hostname}" ]]; then
-				if [[ "${remotes_arg}" != "localhost" ]]; then
+				if [[ -n "${remotes_arg}" ]]; then
```

Comment:
Why won't it fire?    All we are doing is checking each remote in the array  and matching against local interfaces: if it matches, and `remotes_args` is non-empty, then we know that the user specified explicitly a local interface. The first time that we see that, we remember that they did that and set `local_hostname`; subsequently, if it happens again, then we drop it. I think it's good as is, but I already had a headache before I started looking and now it's worse, so who knows? :-)

Reply:
Upon reconsideration, I like the new code better than the previous.  `${remotes_arg}` contains what the user specified, so we only issue the warning(s) if they actually specified something, and otherwise we proceed silently, which is nice (whereas before, we would have skipped the warning if the user specified `localhost`).